platforms/golang/versions/focal-scm/versionsToBuild.txt

```diff
@@ -8,4 +8,5 @@
 1.18.10, 5e05400e4c79ef5394424c0eff5b9141cb782da25f64f79d54c98af0a37f8d49,
 1.19, 464b6b66591f6cf055bc5df90a9750bf5fbc9d038722bb84a9d56a2bea974be6,
 1.19.3, 74b9640724fd4e6bb0ed2a1bc44ae813a03f1e72a4c76253e2d5c015494430ba,
-1.19.5, 36519702ae2fd573c9869461990ae550c8c0d955cd28d2827a6b159fda81ff95,
\ No newline at end of file
+1.19.5, 36519702ae2fd573c9869461990ae550c8c0d955cd28d2827a6b159fda81ff95,
+1.19.7, 7a75720c9b066ae1750f6bcc7052aba70fa3813f4223199ee2a2315fd3eb533d,
```

Comment:
Since app service still uses old images, we should probably add go for stretch as well, right?

Reply:
we did not update Golang stretch version since 1.18. that is why I did not add the new version.

Reply:
checked the email. looks like they are able to get golang 19.5, so i think your change here is fine, and its not needed for stretch... 

side note: i wonder if app service updated their build images to a more recent version. that would be nice as it would unblock some of the work to deprecate stretch